src/datachain/lib/dc.py

```diff
@@ -305,21 +318,23 @@ def add_schema(self, signals_schema: SignalSchema) -> "Self":  # noqa: D102
     @classmethod
     def from_storage(
         cls,
-        path,
+        uri,
         *,
         type: Literal["binary", "text", "image"] = "binary",
         session: Optional[Session] = None,
         in_memory: bool = False,
         recursive: Optional[bool] = True,
         object_name: str = "file",
         update: bool = False,
+        client_config=None,
+        anon: bool = False,
```

Comment:
Add info to docstrings about these two params?

Reply:
Added for `anon` ... `client_config` is being discussed now if we are going to leave it or not